src/_internal/clear/src/Clear.tsx

```diff
@@ -36,7 +36,6 @@ export default defineComponent({
                 <div
                   key="dismiss"
                   class={`${clsPrefix}-base-clear__clear`}
-                  onClick={this.onClear}
                   onMousedown={this.handleMouseDown}
```

Comment:
Maybe just my opinion but I would go for the click rather than the mouse down. The mousedown has no consistent behavior when it comes to touch displays, and many more pitfalls.

Reply:
https://github.com/tusen-ai/naive-ui/commit/5094c407cf220421afa3ef79841f0150e08ed46b
Maybe you should see this commit.

Reply:
Okay, I get the point. 

However, I wonder if there isn't a better alternative in both cases, especially as in both cases the focus should be on the input afterwards.

Thanks anyway 😄